<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>621</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>851</width>
    <height>621</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Phone Validator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>620</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>851</width>
      <height>621</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>850</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 85, 255, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>850</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
font: 16pt &quot;Baskerville Old Face&quot;;</string>
     </property>
     <property name="text">
      <string>VALIDATOR V1.1.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>841</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Baskerville Old Face&quot;;</string>
     </property>
     <property name="text">
      <string>VALIDATE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="status_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>661</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 170, 0);</string>
     </property>
     <property name="text">
      <string>Ready to start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="country_code">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter Country Code</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="service_code">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter the area code</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="length">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter the length of Remaining Digits</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="number">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bookman Old Style</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>How many numbers to generate?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="generate_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>420</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#generate_btn{
background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
border: none;
 padding: 5px;
 border-radius: 25px;
}

QPushButton#generate_btn:hover{
background-color: rgba(0,0,0,0.9)
}</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="validate_btn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>420</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#validate_btn{
background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
border: none;
 padding: 5px;
 border-radius: 25px;
}

QPushButton#validate_btn:hover{
background-color: rgba(0,0,0,0.9)
}</string>
     </property>
     <property name="text">
      <string>Validate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_from_validator">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>0</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#back_from_validator{
background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
border: none;
 padding: 5px;
}

QPushButton#back_from_validator:hover{
background-color: rgba(0,0,0,0.9)
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>420</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#close_btn{
background-color: rgba(0, 85, 255, 208);
color: rgb(255, 255, 255);
border: none;
 padding: 5px;
 border-radius: 25px;
}

QPushButton#close_btn:hover{
background-color: red;
}</string>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
